### DISCUSSION TITLE
Transformation | Group by: Support concatenate calculation to join string values

### DISCUSSION THREAD

**Discussion by huynhsamha** [grafana/grafana#38207] [Dashboards, Panels & Visualizations]
**What would you like to be added**:

I want to concatenate string values in **Transformation > Group by > Calculations**

For example:

![table](https://user-images.githubusercontent.com/24195776/105619322-0cb5b280-5e24-11eb-9732-71d88ddcd5a6.png)

In the picture, we want to group by **(Client, Port)**, for example (MessageService, 9060), and (UploadService, 19xx5). And the list of hosts is concatenated or joined by a separator, such as a tag `<br/>`.

And this is where I want to add the new calculation

![group](https://user-images.githubusercontent.com/24195776/105619366-85b50a00-5e24-11eb-84f9-1e36f3fe0e0b.png)

I don't know there will be any effect on [the current list of calculations](https://grafana.com/docs/grafana/latest/panels/calculations-list/) if we add this one to the transformation tab.

Please review if my issue is workable.

**Comment by zachlipp** [comment]
Is there an update here? Or a suggested workaround?

I see this also mentioned at #24484

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!

**Comment by ptrcnull** [comment]
for anyone also looking for this, the closest issue that i've found is #98755
